auth_mellon_httpclient.c

```diff
@@ -390,7 +390,7 @@ int am_httpclient_get(request_rec *r, const char *uri,
         return HTTP_INTERNAL_SERVER_ERROR;
     }
 
-    res = curl_easy_setopt(curl, CURLOPT_TIMEOUT, timeout);
+    res = curl_easy_setopt(curl, CURLOPT_TIMEOUT, timeout * 1L);
```

Comment:
Could you use a more conventional cast here, i.e. `(long)timeout` instead of using multiplication as a type cast? It is a bit confusing to read.